nym-wallet/package.json

```diff
@@ -5,11 +5,11 @@
   "license": "MIT",
   "scripts": {
     "webpack:dev": "yarn webpack serve",
-    "webpack:build": "yarn webpack",
+    "webpack:prod": "yarn webpack --progress --config webpack.prod.config.js",
     "tauri:dev": "yarn tauri dev",
     "tauri:build": "yarn tauri build",
     "dev": "yarn run webpack:dev & yarn run tauri:dev",
-    "build": "yarn run webpack:build & yarn run tauri:build"
+    "build:prod": "run-s webpack:prod tauri:build"
```

Comment:
Nice :+1: I didn't know about this little npm command. Just what we need

Reply:
:rocket: